crates/ruff_benchmark/benches/red_knot.rs

```diff
@@ -29,15 +57,19 @@ fn get_test_file(name: &str) -> TestFile {
 fn setup_case() -> Case {
     let system = TestSystem::default();
     let fs = system.memory_file_system().clone();
-    let init_path = SystemPath::new("/src/tomllib/__init__.py");
     let parser_path = SystemPath::new("/src/tomllib/_parser.py");
     let re_path = SystemPath::new("/src/tomllib/_re.py");
-    let types_path = SystemPath::new("/src/tomllib/_types.py");
```

Comment:
Why did you inline two of these assignments but not the other two?

Reply:
The other paths are used after writing the file. A single variable with the path then seems preferable

Reply:
it seems marginally cleaner to me to assign them all beforehand rather than having some of them assigned beforehand and some inlined. but no strong opinion.